origin.spec

```diff
@@ -7,15 +7,15 @@
 %global import_path github.com/openshift/origin
 
 %global golang_version 1.12
-# %commit and %os_git_vars are intended to be set by tito custom builders provided
-# in the .tito/lib directory. The values in this spec file will not be kept up to date.
+
+# commit and os_git_vars are intended to be set by the build system.
 %{!?commit:
-%global commit 86b5e46426ba828f49195af21c56f7c6674b48f7
+%global commit e795244084
 }
 %global shortcommit %(c=%{commit}; echo ${c:0:7})
 # os_git_vars needed to run hack scripts during rpm builds
 %{!?os_git_vars:
-%global os_git_vars OS_GIT_VERSION='' OS_GIT_COMMIT='' OS_GIT_MAJOR='' OS_GIT_MINOR='' OS_GIT_TREE_STATE=''
+%global os_git_vars OS_GIT_COMMIT='e795244084' OS_GIT_TREE_STATE='dirty' OS_GIT_VERSION='v4.1.0+e795244-1411-dirty' OS_GIT_MAJOR='4' OS_GIT_MINOR='1+' OS_GIT_PATCH='0' KUBE_GIT_MAJOR='1' KUBE_GIT_MINOR='14+' KUBE_GIT_COMMIT='e8fcd6a' KUBE_GIT_VERSION='v1.14.0+e795244084' ETCD_GIT_VERSION='v3.3.10' ETCD_GIT_COMMIT='27fc7e2' 
```

Comment:
any reason why we added the version in too ? 
i may miss something here but if i remember in the past the hack/* script was dynamically working out the version ... not sure if this breaks OCP

Reply:
Afaict this is just what OCP passes as os_git_vars right now and line just hasn't been kept up to date before. Putting them all in one var does obscure things here a bit.

Reply:
_line just hasn't been kept up to date before_ => that is what i was afraid you will say after i posted my comment, thanks for the info !